language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -325,3 +369,84 @@ callProcessSilent cmd args = do
       hPutStrLn stderr $ unlines ["stdout:", out]
       hPutStrLn stderr $ unlines ["stderr: ", err]
       exitFailure
+
+writeTsConfig :: FilePath -> IO ()
+writeTsConfig dir = writeFileUTF8 (dir </> "tsconfig.json") $ unlines
+    [ "{"
+    , "  \"compilerOptions\": {"
+    , "    \"target\": \"es5\","
+    , "    \"lib\": ["
+    , "      \"dom\","
+    , "      \"es2015\""
+    , "     ],"
+    , "    \"strict\": true,"
+    , "    \"noUnusedLocals\": true,"
+    , "    \"noUnusedParameters\": false,"
+    , "    \"noImplicitReturns\": true,"
+    , "    \"noFallthroughCasesInSwitch\": true,"
+    , "    \"outDir\": \"lib\","
+    , "    \"module\": \"commonjs\","
+    , "    \"declaration\": true,"
+    , "    \"sourceMap\": true"
+    , "    },"
+    , "  \"include\": [\"src/**/*.ts\"],"
+    , "}"
+    ]
+
+writePackageJson :: FilePath -> String -> IO ()
+writePackageJson dir name = writeFileUTF8 (dir </> "package.json") $ unlines
+        ["{"
+        , "  \"private\": true,"
+        , "  \"name\": \"" <> name <> "\","
+        , "  \"version\": \"0.13.53\","
+        , "  \"description\": \"Produced by daml2ts\","
+        , "  \"license\": \"Apache-2.0\","
+        , "  \"dependencies\": {"
+        , "    \"@daml/types\": \"0.13.53\","
+        , "    \"@mojotech/json-type-validation\": \"^3.1.0\""
+        , "  },"
+        , "  \"scripts\": {"
+        , "    \"build\": \"tsc --build\","
+        , "    \"build:watch\": \"tsc --build --watch\","
+        , "    \"test\": \"true\","
+        , "    \"lint\": \"eslint --ext .js,.ts src/ --max-warnings 0\""
+        , "  },"
+        , "  \"eslintConfig\": {"
```

Comment:
Do we need all of this? I would expect that `devDependencies` don’t matter and probably a lot of the scripts also don’t matter.

Reply:
I haven't attempted to trim it. I just followed Martin's suggestion and imported it wholesale from davl

Reply:
Sounds reasonable, then let’s keep it.

Reply:
@martin-drhu-da let me know what you think about this and i'll follow up!

Reply:
I suggest we run the linter here too. This way we gain some confidence we generate good TS.

Reply:
We should also consider moving the linter config in a separate file, I think it's called `.eslintrc.json`.

Reply:
@shayne-fletcher Here's a PR demoing the target state on DAVL: https://github.com/digital-asset/davl/pull/202